web/book/src/tutorial/aggregation.md

```diff
@@ -1,95 +1,98 @@
 # Aggregation
 
 A key feature of analytics is reducing many values down to some summary. This
-act is called aggregation and always includes an aggregation function, for
-example `average` or `sum`:
+act is called "aggregation" and always includes a function &mdash; for
+example, `average` or `sum` &mdash; that reduces the table to a single row.
 
-<!-- aggregating ids does not make much sense, should we switch to some other table? -->
+**aggregate:** The `aggregate` transform takes a tuple to create one or many new columns
+that "distill down" data from all the rows.
 
 ```
-from albums
-aggregate {sum_of_ids = sum album_id}
+from invoices
+aggregate { sum_of_orders = sum total }
 ```
+In the example above, the `total` column of all rows of the `invoices` table
+is summed to produce a single value.
 
-`aggregate` is quite similar to `derive` and `select` as it is creating a new
-column, with the option to assign it a name. Just as select, it will discard all
-columns that are not present in the tuple, but with a key difference: it will
-always produce a single row.
-
-Just as `select` (and actually `derive` too), `aggregate` takes a tuple of
-aggregation expressions, which means that you can produce multiple summaries at
-once:
+`aggregate` can produce multiple summaries at once when
+one or more aggregation expressions are contained in a tuple.
+`aggregate` discards all columns that are not present in the tuple.
 
 ```
-from albums
+from invoices
 aggregate {
-    sum_of_ids = sum album_id,
-    mean_id = average album_id,
+    num_orders = count this,
+    sum_of_orders = sum total,
 }
 ```
 
-Here we've also split the tuple over multiple lines to make it more readable.
-Obviously, you can do this with `select` and `derive` too.
+In the example above, the result is a single row with two columns.
+The `count` function displays the number of rows in the table that was passed in;
+the `sum` function adds up the values of the `total` column of all rows.
 
 ## Grouping
 
-Say that now we want to produce summaries of albums _for each artist
-separately_. We cannot do this, as `aggregate` will always produce a single row.
+Suppose we want to produce summaries of invoices _for each city_
+in the table.
+`aggregate` cannot do this because it will always produce a single row.
```

Comment:
This could be rephrased away from the negative (particularly since it's not exactly correct — it can do it — it just requires a `group`

Reply:
I agree. I'll rephrase as "We could have a series of aggregates - one for each city..."